client/main.lua

```diff
@@ -39,6 +39,32 @@ local function safeAnim()
     TaskPlayAnim(cache.ped, 'amb@prop_human_bum_bin@idle_b', 'exit', 8.0, 8.0, -1, 50, 0, false, false, false)
 end
 
+local function checkInteractStatus(register)
+    if sharedConfig.registers[register].robbed then
+        return false
+    end
+
+    local leoCount = lib.callback.await('qbx_storerobbery:server:leoCount', false)
+    if leoCount > sharedConfig.minimumCops then
+        return true
+    end
```

Comment:
Is this going to cause an issue being in the canInteract function? Not sure if that is run in a loop or not. If it is going to be an issue, check this on select rather than canInteract

Reply:
I don't see why it would be an issue